examples/authorized_api/authorization.go

```diff
@@ -18,6 +67,54 @@ func MustAuthorize(handler http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+func ZebedeeMustAuthorize(handler http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		status, err := zebedeeValidateAuth()
+		if err != nil {
+			if status == "401 Unauthorized" {
```

Comment:
See comment about using the status code int instead of the string value.

Reply:
done